jibri/rootfs/defaults/jibri.conf

```
@@ -108,4 +108,12 @@ jibri {
       enable-stats-d = {{ .Env.ENABLE_STATS_D }}
     }
     {{ end -}}
-}
\ No newline at end of file
+
+    {{ if .Env.ALL_MUTED_TIMEOUT -}}
+    call-status-checks {
+        // If all clients have their audio and video muted, Jibri consideres this
+        // as an empty call and stops the recording after ALL_MUTED_TIMEOUT expires.
+        all-muted-timeout = "{{ .Env.ALL_MUTED_TIMEOUT }}"
+    }
+    {{ end -}}
```

```suggestion
    call-status-checks {
        {{ if .Env.NO_MEDIA_TIMEOUT -}}
        // If all clients have their audio and video muted and if Jibri does not
        // detect any data stream (audio or video) comming in, it will stop
        // recording after NO_MEDIA_TIMEOUT expires.
        no-media-timeout = {{ .Env.NO_MEDIA_TIMEOUT }}
        {{ end -}}
    
        {{ if .Env.ALL_MUTED_TIMEOUT -}}
        // If all clients have their audio and video muted, Jibri consideres this
        // as an empty call and stops the recording after ALL_MUTED_TIMEOUT expires.
        all-muted-timeout = {{ .Env.ALL_MUTED_TIMEOUT }}
        {{ end -}}

        {{ if .Env.DEFAULT_CALL_EMPTY_TIMEOUT -}}
        // When detecting if a call is empty, Jibri takes into consideration for how
        // long the call has been empty already. If it has been empty for more than
        // DEFAULT_CALL_EMPTY_TIMEOUT, it will consider it empty and stop the recording.
        default-call-empty-timeout = {{ .Env.DEFAULT_CALL_EMPTY_TIMEOUT }}
        {{ end -}}
    }
```

Shall we add them all in incase it is `no-media-timeout`?

---

yeah, probably may as well I guess